airbyte-integrations/connectors/destination-csv/src/test-integration/java/io/airbyte/integrations/destination/csv/CsvDestinationAcceptanceTest.java

```diff
@@ -90,4 +130,20 @@ protected void tearDown(final TestDestinationEnv testEnv) {
     // no op
   }
 
+  public class CSVDataArgumentsProvider extends DataArgumentsProvider {
+
+    public CSVDataArgumentsProvider(){};
+    @Override
+    public Stream<? extends Arguments> provideArguments(final ExtensionContext context) {
+      ProtocolVersion protocolVersion = getProtocolVersion();
```

Comment:
the method `io.Airbyte.integrations.standardtest.destination.argproviders.util.ArgumentProviderUtil.getProtocolVersion` should be used with test context as a param. So it will looks like:
```
  public Stream<? extends Arguments> provideArguments(final ExtensionContext context) throws Exception {
    ProtocolVersion protocolVersion = getProtocolVersion(context);
```

Reply:
Thanks for the pointer @suhomud! Could you mention `ArgumentProviderUtil.getProtocolVersion` in the comment for `DestinationAcceptanceTest.getProtocolVersion`? The current comment says something about reflection, but it does not mention where we use reflection and it also does not provide guidance on how to use `getProtocolVersion`